guides/doc-Project_API/topics/con_api-cheat-sheet.adoc

```diff
@@ -0,0 +1,23 @@
+[id="api-cheat-sheet"]
+= API cheat sheet
+
+You can review the following examples of how to use the {ProjectName} API to perform various tasks.
+You can use the API on {ProjectServer} via HTTPS on port 443 or on {SmartProxyServer} via HTTPS on port 8443.
```

Comment:
Needs technical ACK please.

Reply:
API calls through SmartProxy:8443 are deprecated from Foreman 3.12 (checked with SMEs). However, that will require larger changes than just this line...

Reply:
Or should we just add a deprecation note for now and deal with the rest once it's removed?